File: content/problems/1876-substrings-of-size-three-with-distinct-characters/solutions/fixed-window-comparison/Solution.kt
class Solution { fun countGoodSubstrings(s:String):Int{var answer=0;for(i in 0..s.length-3)if(s[i]!=s[i+1]&&s[i]!=s[i+2]&&s[i+1]!=s[i+2])answer++;return answer} }

File: content/problems/1876-substrings-of-size-three-with-distinct-characters/solutions/fixed-window-comparison/solution.cpp
class Solution {
    public: int countGoodSubstrings(string s){
        int answer=0;
        for(int i=0;i+2<(int)s.size();++i)if(s[i]!=s[i+1]&&s[i]!=s[i+2]&&s[i+1]!=s[i+2])++answer;
        return answer;
    }
};
